doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,72 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron holography data:
+ 
+* :py:class:`~._signals.hologram_image.HologramImage`
+* :py:class:`~._signals.wave_image.WaveImage`
+
+Both inherit directly from the :py:class:`~._signals.image.Image` class and thus can use all of its
+functionality. The usage of both classes is explained in the following sections.
+
+
+The HologramImage class
+=======================
+
+The :py:class:`~._signals.hologram_image.HologramImage` class is designed to hold images acquired via
+electron holography. The complex electron wave can be reconstructed from the signal via  the
+:py:func:`~._signals.hologram_image.HologramImage.reconstruct_wave_image` method which will return
+a :py:class:`~._signals.wave_image.WaveImage` class, which can then be further analysed.
+
+To transform a :py:class:`~._signals.image.Image` (or subclass) into a 
+:py:class:`~._signals.hologram_image.HologramImage` use:
+
+.. code-block:: python
+
+    >>> im.set_signal_type('HOLOGRAM')
+
+
+The WaveImage class
+===================
+
+The :py:class:`~._signals.wave_image.WaveImage` class can hold information about the complex electron
+wave. As such, relevant properties like the `amplitude`, `phase` and the `real` and `imag` part can be
+directly accessed and return appropriate :py:class:`~._signals.image.Image` signals.
+
+
+Unwrap the phase
+----------------
+
+With the :py:func:`~._signals.wave_image.WaveImage.get_unwrapped_phase` method the phase can be
+unwrapped and returned as an :class:`~hyperspy._signals.Image`. The underlying method is

```

Comment:
Thanks :-), I already corrected this in my local copy, but currently I am not able to push :-/...
Not sure what went wrong, I tried to also split off the HologramImage class into a separate PR. I'm locally using Mercurial, maybe the transfer to Git is problematic (the Git branches are bookmarks in Mercurial, which is quite strange)...